contracts/gateway/OSTPrime.sol

```diff
@@ -49,20 +49,20 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig, Mutex {
 
     using SafeMath for uint256;
 
-    /** Emitted whenever OST Prime token is converted to OST Prime base token. */
-    event TokenUnwrapped(
+    /** Emitted whenever OST is converted to coin. */
+    event OSTUnwrapped(
```

Comment:
`TokenUnwrapped` is still correct, because the ERC20 token is unwrapped to become a base coin.  Please update the comment along the same line